boxoffice/models/line_item.py

```diff
@@ -24,6 +24,25 @@ def make_ntuple(item_id, base_amount, **kwargs):
         kwargs.get('discounted_amount', Decimal(0)))
 
 
+class Assignee(BaseMixin, db.Model):
+    __tablename__ = 'assignee'
+    __table_args__ = (db.UniqueConstraint('item_collection_id', 'email'),)
```

Comment:
Add a `user_id` column linking this to a Lastuser user. Users should be able to edit their own ticket details and to initiate transfers.